src/test/java/se/sundsvall/notes/api/NoteResourceFailuresTest.java

```diff
@@ -362,7 +447,7 @@ void getNotesByPartyIdInvalidPartyId() {
 		// Parameter values
 		final var partyId = "invalid";
 
-		final var response = webTestClient.get().uri(builder -> builder.path("/notes").queryParam("partyId", partyId) .build())
+		final var response = webTestClient.get().uri(builder -> builder.path(PATH).queryParam("partyId", partyId).queryParam("municipalityId", MUNICIPALITY_ID) .build())
```

Comment:
As municipalityId is now part of body instenad of path, maybe test that get-query without municipalityId renders an exception?